go.mod

```diff
@@ -3,26 +3,32 @@ module github.com/gardener/gardener
 go 1.13
```

Comment:
Nit: `make revendor` updates some minor changes under `vendor/`. Is it also the same for your?